USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -336,6 +355,50 @@ private string RenderMarker(Marker input)
                 case FVMarker fVMarker:
                     output.AppendLine($"<span class=\"versemarker\">{fVMarker.VerseCharacter}</span>");
                     break;
+                case TRMarker tRMarker:
+                    output.AppendLine("<div>");

```

Comment:
I think this is wrong. We don't want to start a new table for every row. We're going to have to be a little creative of how we handle these tables. Craig has a couple of ideas.

Reply:
Yeah I understand. I'll talk to Craig about the possible solutions. Thanks @rbnswartz

Reply:
I created a TableBlock marker to contain the adjacent table information. @PurpleGuitar @rbnswartz